src/CommunityToolkit.Maui/Behaviors/Validators/EmailValidationBehavior.shared.cs

```diff
@@ -37,6 +37,18 @@ protected override async ValueTask<bool> ValidateAsync(string? value, Cancellati
 		return IsValidEmail(value) && await base.ValidateAsync(value, token);
 	}
 
+	/// <inheritdoc /> 
+	protected override void OnAttachedTo(VisualElement bindable)
+	{
+		// Assign Keyboard.Email if the user has not specified a specific Keyboard layout
+		if (bindable is InputView inputView && inputView.Keyboard == Keyboard.Default)
```

Comment:
do we need to return previous keyboard on detach?

Reply:
I think that's fair. Added ✅ 

I could see a concern if a developer explicitly set `Keyboard.Email` then detached `EmailValidationBehavior`. But I imagine that's a rare edge case.